### DISCUSSION TITLE
Multi BMS Support

### DISCUSSION THREAD

**Discussion by gutowscr471** [ai-republic/bms-to-inverter#15] [General]
I have a EG6000xp inverter made by LuxPower which which read Pylon CAN. I have several of the older non-inverter BMS connected using voltage for charging but would like to use SOC. Will this aggregate the multiple BMS into one overall SOC and other parameters to the inverter via Pylon protocol?

**Comment by ai-republic** [comment]
Yes sure, it supports multiple BMSes. Even different brands and protocols. All values will be aggregated and sent to the inverter in the inverter specific protocol.
